lib/cdo/hamburger.rb

```diff
@@ -94,7 +94,7 @@ def self.get_hamburger_contents(options)
 
     teacher_entries = [
       {title: "my_dashboard", url: CDO.studio_url("/home")},
-      {title: "course_catalog", url: CDO.studio_url("/courses?view=teacher")},
+      {title: "course_catalog", url: CDO.code_org_url("/teach")},
```

Comment:
Will want to sync with @megcrenshaw on this one. They were working on this before their vacation. We are updating this text to Curriculum Catalog and it should go to the Curriculum Catalog link

Reply:
Sounds good! I'll update the URL for now to /catalog and we can just override the text later based on the spec Meg is working on

Reply:
I also changed it to /catalog in my PR, so there shouldn't be any merge conflicts now!